nbdime/diffing/notebooks.py

```diff
@@ -194,6 +195,8 @@ def diff_mime_bundle(a, b, path=None):
     "/cells/*": diff,
     "/cells/*/outputs": diff_sequence_multilevel,
     "/cells/*/outputs/*": diff_single_outputs,
+    "/cells/*/attachments": diff_sequence_multilevel,
+    "/cells/*/attachments/*": diff_mime_bundle,
```

Comment:
I started working on it, and a major question appeared: If someone has renamed an attachment (identical or short edit of content, but different key), how can we represent this? For normal notebook content this is maybe not so relevant, but this would be likely for attachments. Should we introduce a `rename` diff op?